dags/vds_pull_vdsdata.py

```diff
@@ -135,8 +129,12 @@
             op_kwargs = conns
         )
 
-        skip_update_inventories >> [pull_detector_inventory_task, pull_entity_locations_task]
+        t_done = ExternalTaskMarker(
+                task_id="done",
+                external_dag_id="vds_pull_vdsvehicledata",
+                external_task_id="starting_point"
+        )
+
+        [pull_detector_inventory_task, pull_entity_locations_task] >> t_done
```

Comment:
This means the downstream DAG would be triggered when this task group succeeds not the whole DAG. Just confirming this's what you meant